src/pages/Countries.tsx

```diff
@@ -0,0 +1,88 @@
+import CountrySmall from "../components/CountrySmall";
+import { useEffect, useState } from "react";
+import { Country } from "../types/CountrySmallProps";
+
+const Countries = () => {
+  const [countries, setCountries] = useState([]);
+  const [error, setError] = useState("");
+  const [regionFilter, setRegionFilter] = useState("");
+  const [searchInput, setSearchInput] = useState("");
+
+  useEffect(() => {
+    fetchCountries("all");
+  }, []);
+
+  const fetchCountries = async (url: string, prevURL?: string) => {
+    if (!prevURL) prevURL = "";
```

Comment:
use default value instead of optional
```suggestion
  const fetchCountries = async (url: string, prevURL: string = "") => {
```

Reply:
Done